cupy/_manipulation/kind.py

```diff
@@ -2,6 +2,31 @@
 from cupy import _core
 
 
+def asarray_chkfinite(a, dtype=None, order=None):
+    """
+    Convert the given input to an array,
+    and raises an error if the input contains NaNs or Infs.
```

Comment:
CuPy's convention is to start the short description right after the triple quotes. Also, correct the verb to be consistent:
```suggestion
    """Converts the given input to an array,
    and raises an error if the input contains NaNs or Infs.
```